docs/howto/configure/auth-management.md

```diff
@@ -85,7 +84,6 @@ Presently, this involves a few more manual steps than the `auth0` setup describe
           hub:
             config:
               GitHubOAuthenticator:
-                oauth_callback_url: https://{{ HUB_DOMAIN }}/hub/oauth_callback
```

Comment:
Hmm, I'm not sure I understand why this is being removed? sorry :blush:

Reply:
It gets moved to the section below that _won't_ be `sops` encrypted. I don't see a need for the hub oauth callback URL to be encrypted.

Reply:
Gets added again here https://github.com/2i2c-org/infrastructure/pull/1010/files#diff-93bc32e20dd98e9580a554df1df633361f26bf66395672281e4643a9711bb06bR140

Reply:
Oh, makes sense now! Thanks